### PR TITLE
Add upload file functionality

### PR DESCRIPTION
**Pull request by marks**
@johnjoo1 - could you take a look and confirm this works? There's an example in the `examples/` dir.

### CODE REVIEW

File: examples/upload_file.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# connect to domino; be sure to have these environment variables set
+#  (runs inside a Domino executor automatically set these for you)
+domino = Domino("marks/test", api_key=os.environ['DOMINO_USER_API_KEY'], host=os.environ['DOMINO_API_HOST'])
+
+f = open('numbers.csv', 'rb')
+r = domino.files_upload("/a_new_folder/a_new_file.csv", f)
+
+if r.status_code == 201:
+	print "Upload successful"
```
Comment (johnjoo1) [inline examples/upload_file.py]: should make this compatible with Python 3.

File: examples/upload_file.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+else:
+	print "!! Upload failed"
```
Comment (johnjoo1) [inline examples/upload_file.py]: same here

**Comment by marks** [comment]
@johnjoo1 can you take a look at this and agree if it can be merged? It technically will not work until we address PR #6 for py3 compat but I want to do one thing at a time

**Comment by marks** [comment]
Spoke with John... merging

**Pull request merged**